Updating Outdated Mac Support (#2524)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Skype/SfbServer/plan-your-deployment/clients-and-devices/meetings-clients.md

---

# Plan for Meetings clients (Web App and Meetings App)
  
 >[!IMPORTANT]

>  >As of April 5th, 2019, the Skype for Business Web App and Skype Meetings App will be replaced by the Skype for Business desktop app for Mac users. For more information, please see [Skype for Business desktop app on Mac to replace Skype Meetings App (web)](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Skype-for-Business-desktop-app-on-Mac-to-replace-Skype-Meetings/ba-p/357958).

Reviewer: ```suggestion
 >On April 5th, 2019, the Skype for Business Web App and Skype Meetings App have been replaced by the Skype for Business desktop app for Mac users. For more information, please see [Skype for Business desktop app on Mac to replace Skype Meetings App (web)](https://techcommunity.microsoft.com/t5/Skype-for-Business-Blog/Skype-for-Business-desktop-app-on-Mac-to-replace-Skype-Meetings/ba-p/357958).
```

Reviewer 2: On....have been?  Suggest review is reviewed
